addons/project/static/src/components/project_right_side_panel/components/project_profitability.xml

```diff
@@ -78,22 +78,23 @@
                 <thead>
                     <tr class="align-top">
                         <th>Margin</th>
+                        <t t-set="total_revenue" t-value="revenues.total.to_invoice + revenues.total.invoiced"/>
                         <th class="text-end" t-att-class="margin.invoiced_billed &lt; 0 ? 'text-danger' : 'text-success'">
                             <t t-esc="props.formatMonetary(margin.invoiced_billed)"/><br/>
-                            <t t-if="costs.total.billed != 0">
-                                <t t-esc="margin.invoiced_billed > 0 ? '+' : ''"/><t t-esc="(margin.invoiced_billed / (-costs.total.billed) * 100).toFixed(0)"/>%
+                            <t t-if="total_revenue != 0">
+                                <t t-esc="margin.invoiced_billed > 0 ? '+' : ''"/><t t-esc="(margin.invoiced_billed / total_revenue * 100).toFixed()"/>%
                             </t>
                         </th>
                         <th class="text-end" t-att-class="margin.to_invoice_to_bill &lt; 0 ? 'text-danger' : 'text-success'">
                             <t t-esc="props.formatMonetary(margin.to_invoice_to_bill)"/><br/>
-                            <t t-if="costs.total.to_bill != 0">
-                                <t t-esc="margin.to_invoice_to_bill > 0 ? '+' : ''"/><t t-esc="(margin.to_invoice_to_bill / (-costs.total.to_bill) * 100).toFixed(0)"/>%
+                            <t t-if="total_revenue != 0">
+                                <t t-esc="margin.to_invoice_to_bill > 0 ? '+' : ''"/><t t-esc="(margin.to_invoice_to_bill / total_revenue * 100).toFixed()"/>%
```

Comment:
Not my expertise, so I'm not entirely sure, I calculated it relative to total revenue because it allows you to add up the percentages horizontally:

![proft3](https://github.com/odoo/odoo/assets/13054393/05fc5791-d432-4175-826b-539a3265c998)

While I don't find the percentages in the other method particularly helpful (but I'm also not an expert, so...):

![profit2](https://github.com/odoo/odoo/assets/13054393/34471405-2d07-4e6f-a6d9-5c26ac3127bb)

According to chatGPT, the latter is more conventional, but it's hard to find any online resources on this specific case, so it could be making shit up. Will change it though.